### ISSUE TITLE
Organizing the Project

### ISSUE THREAD

**Issue by scott-wilson** [vfx-rs/organization#3]
There's a few ways we can organize the vfx-rs project. But, before we look at possible solutions, there's some questions.

- What are the group's goals?
- How do we handle policies? If a new or updated policy is proposed, how should we balance between giving the proposal time for discussion, and making sure the proposals don't stagnate.
- How do we decide if a policy is accepted or not? Do we leave it votes, or have a small group that has final decision (and how is that group created)?

**Comment by luke-titley** [comment]
_What are the group's goals?_
* To wrap the foundational vfx libs in rust.
* To produce c apis for the libs we wrap (if they don't already have them).

**Comment by scott-wilson** [comment]
Any concern with me making a folder in the org repo for RFCs? Worst case, we can split this out later.

Also, I'd like to add one additional goal (for bus factor):

- Give ownership of wrappers and crates to the wrapped library owners, or the ASWF.

**Comment by luke-titley** [comment]
No concerns from me, but having an rfc repo seems cleaner to me, and isn't any harder to do.
If you're concerned about it being discovered you can always link to it from the README.md of organization.

**Comment by scott-wilson** [comment]
Sounds good. I'll create an rfc repo and add some of the mentioned items to it.

**Comment by Andlon** [comment]
Sorry for the somewhat only tangentially related post, but I wanted to say that as a "self-appointed representative" of a small group of computer graphics researchers from different universities, we're following with interest what you guys are doing here. We were just discussing wrapping some of the VFX libraries (USD/Alembic in particular), and were pleasantly surprised to stumble upon your project, which seems to be in its infancy.

Some background: we're all researchers in physics-based animation. That is, we develop new algorithms for simulating physical phenomena applicable to animation/vfx, virtual prototyping, interactive physics simulation and so on. Between us, we have at this point 2 published papers [1] built with Rust (we are working towards open sourcing our code here), and a third one is currently awaiting peer review. We are therefore getting increasingly invested in Rust, and a crucial part of our pipeline is getting the simulation data from our simulator to VFX software like Maya or Blender for visualization. This is presumably where our interests overlap. So far we've been using some home-baked solutions involving sequences of PLY files and plugins in Maya/Blender to import simulation data, but this is less than ideal. We're therefore looking at a solution involving USD/Alembic instead.

To summarize, we're very interested in your efforts here, and we may be interested in contributing in one way or another. Are you planning to use, or interested in, an asynchronous messaging platform like Discord?

[1] See https://animation.rwth-aachen.de/publication/0572/ and https://animation.rwth-aachen.de/publication/0571/

**Comment by luke-titley** [comment]
Hey @Andlon there's a rust channel on awsf slack, feel free to join to discussion.
https://slack.aswf.io/
